lib/eslint-patch.js

```diff
@@ -1,33 +1,31 @@
-'use strict';
-var meld = require('meld');
-var docs = require('./docs');
-var Config = require("eslint/lib/config");
-var ConfigUpgrader = require('./config_upgrader');
-
-var supportedPlugins = ['react', 'babel'];
-
-module.exports = function patcher(eslint) {
-
-  meld.around(eslint.CLIEngine, 'loadPlugins', function(joinPoint) {
```

Comment:
this was removed because this hook doesn't ever actually fire? (I think I saw another comment you made about this)